compute_sdk/tests/unit/test_mpi_functions.py

```diff
@@ -7,3 +8,18 @@ def test_mpi_function(run_in_tmp_dir):
     bash_result = mpi_func()
 
     assert "$PARSL_MPI_PREFIX hostname" in bash_result.cmd
+
+
+def test_mpi_function_name_default(run_in_tmp_dir):
+    """Confirm that the MPIFunction defaults to MPIFunction"""
+    mpi_func = MPIFunction("hostname")
+
+    assert mpi_func.__name__ == type(mpi_func).__name__
+
+
+@pytest.mark.parametrize("fn_name", ["lammps", "my_mpi_app"])
+def test_mpi_function_name(run_in_tmp_dir, fn_name):
+    """Confirm that the MPIFunction defaults to MPIFunction"""
+    mpi_func = MPIFunction("hostname", name=fn_name)
```

Comment:
The second docstring isn't valid.  However, given that the test names have already identified what the tests are purportedly testing, I suggest that the docstrings aren't necessary here.